frontend/src/main.js

```diff
@@ -16,10 +77,12 @@ function getRandom(min, max) {
 // ai = 0 two humans
 // (to be implemented) ai = -2 four humans
 window.startGame = (ai) => {
-    let gameResult = "Game is not finished yet"
-    let maxScore = 11
-    let hight = 20;
-    let width = 40;
+    let gameResult = "Game is not finished yet";
+    let isPaused = false;
+    const maxScore = 5;
+    const playerSpeed = 17;  //12
+    
+    // create a scene and camera
```

Comment:
these could be moved outside, maybe have some initialisation to them in a static like manner=> if they havent been created create them

Reply:
We might want to move it to the state json that we elaborated together in order to add it to the settings later.